rust/kcl-lib/src/execution/exec_ast.rs

```diff
@@ -1924,7 +1931,16 @@ fn type_check_params_kw(
     args: &mut crate::std::args::KwArgs,
     exec_state: &mut ExecState,
 ) -> Result<(), KclError> {
+    let mut arg_errors = Vec::new();
     for (label, arg) in &mut args.labeled {
+        if label.starts_with("__ERR") {
```

Comment:
Maybe instead of putting these strings into the args (which then relies on some other code to check for this prefix) we could capture the idea "this might be a label, or an error where a label is expected" in types, as an enum or something? that way callers have to match them, and the string "_ERR" can't accidentally make its way to the user, like in the example I pasted.

Reply:
Heh, yeah, I was being lazy :-(

Reply:
Addressed by keeping the error args in a different part of KwArgs